src/Controller/BackendCsvImportController.php

```diff
@@ -0,0 +1,243 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Controller;
+
+use Contao\CoreBundle\Exception\CsvImportErrorException;
+use Contao\CoreBundle\Exception\RedirectResponseException;
+use Contao\CoreBundle\Exception\ResponseException;
+use Contao\CoreBundle\Framework\ContaoFrameworkInterface;
+use Contao\DataContainer;
+use Contao\FileUpload;
+use Doctrine\DBAL\Connection;
+use Contao\CoreBundle\Util\CsvImportUtil;
+use Symfony\Component\HttpFoundation\Cookie;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+
+/**
+ * Handles the backend CSV import.
+ *
+ * @author Kamil Kuzminski <https://github.com/qzminski>
+ */
+class BackendCsvImportController
```

Comment:
Controllers in Symfony use `Action` as method suffix. I think they should all use that too here.